pkg/config/config.go

```diff
@@ -879,7 +871,26 @@ func InitConfig(config Config) {
 
 	// Process agent
 	config.SetKnown("process_config.dd_agent_env")
-	config.SetKnown("process_config.enabled")
+	config.BindEnvAndSetDefault("process_config.process_collection.enabled", false, "DD_PROCESS_COLLECTION")
+	config.BindEnvAndSetDefault("process_config.container_collection.enabled", true, "DD_CONTAINER_COLLECTION")
```

Comment:
We may want to pick different env var names for these, suggestions:
- `DD_PROCESS_AGENT_COLLECT_PROCESSES`
- `DD_PROCESS_AGENT_COLLECT_CONTAINERS`

Reply:
I think viper will allow you to bind two environment variables to one setting, so that should work :)